finalPrep/annotations.js

```diff
@@ -1,42 +1,76 @@
 // Add your annotations, line by line, to the code below using code comments.
 // Use the double // to create a new comment
 
-//Build a Bear
+//Build a Bear class with a method (function to buildABear)
+//function defined in camelCase to build a bear declared with dynamic properties/start of curly bracket defining js statement
 function buildABear(name, age, fur, clothes, specialPower) {
+//the variable greeting is declared with a pre-written greeting including an interpolation to print a question including the name from the function.
   var greeting = `Hey partner! My name is ${name} - will you be my friend?!`;
+  //the variable demographics is defined with dynamic properties for name and age
   var demographics = [name, age];
+  //the variable powerSaying is declared with a concatenation to combine a string with another variable declared
   var powerSaying = "Did you know that I can " + specialPower + " ?";
+//the variable builtBear is defining an object with kay value pairs (mostly dynamic values,
+//a few static values, and a combination, mixed with a string within an array)
   var builtBear = {
+//properties key-value pair (dynamic)
     basicInfo: demographics,
+    //propteries key-value pair (dynamic)
     clothes: clothes,
+    //properties key-value pair (dynamic)
     exterior: fur,
+    //properties key-value pair (static)
     cost: 49.99,
+    //properties key-value pair (dynamic with a string in an array)
     sayings: [greeting, powerSaying, "Goodnight my friend!"],
+    //properties key-value pair (static) with a boolean property of true
     isCuddly: true,
+//end of curly bracked that declared the function
   };
-
+//the return statement ends this function and states the value to be returned (had to look this up)
   return builtBear
+  //start of new curly bracket for calling the function
 }
+// Two object instances being called of the function above and because of dynamic values (name, age, fur, clothes, specialPower),
+//the output will vary by the object instances called (in this case the two bears
+//are with unique details to identify them apart)
 
 buildABear('Fluffy', 4, 'brown', ['pants', 'jorts', 'tanktop'], 'give you nightmares');
 buildABear('Sleepy', 2, 'purple', ['pajamas', 'sleeping cap'], 'sleeping in');
 
 
 
-//FizzBuzz
+//FizzBuzz program declared with the function fizzBuzz with the parameters of the function within ()
+// and function statements have {around it}
 function fizzBuzz(num1, num2, range) {
+  //this is a for loop with an if else statment to answer the function being called below
   for (var i = 0; i <= range; i++) {
+// first if statement with a specific console output if the condition within is met or the code will continue to run
```

Comment:
If I was looking through code annotations, I'd be curious _what_ the particular condition is. In the as you wait for Turing to start, you can do some digging into logical operators, or what a modulo is (the % symbol).